docs/resources/elb_loadbalancer.md

```diff
@@ -155,6 +155,12 @@ The following arguments are supported:
 
 * `auto_renew` - (Optional, String) Specifies whether auto renew is enabled. Valid values are **true** and **false**.
 
+* `autoscaling_enabled` - (Optional, Bool) Specifies whether autoscaling is enabled. Valid values are **true** and
+  **false**.
+
+* `min_l7_flavor_id` - (Optional, String) The min L7 flavor id of ELB. The type is L7_Elastic. It can not be empty when
+  the loadbalancer has a listener of L7.
```

Comment:
```suggestion
* `min_l7_flavor_id` - (Optional, String) Specifies the ID of the minimum Layer-7 flavor for elastic scaling.
  This parameter cannot be left blank if there are HTTP or HTTPS listeners.
```
this is copied from [API dosc](https://support.huaweicloud.com/intl/en-us/api-elb/CreateLoadBalancer.html#CreateLoadBalancer__request_CreateLoadbalancerAutoscalingOption), you can also add more informations about it.

Reply:
done